<launch>
  <!-- declare arg to be passed in -->
  <arg name="image_topic" default="/camera/rgb/image_raw"/>

  <!-- Nodes -->
  <!--<node name="ros_tld_gui_node" pkg="tld_tracker" type="tld_gui_node">-->
  <!--<node launch-prefix="gnome-terminal -e" name="ros_tld_gui_node" pkg="tld_tracker" type="tld_gui_node" output="screen" required="true">-->
  <node name="ros_tld_gui_node" pkg="tld_tracker" type="tld_gui_node" required="true">
    <remap from="fps_tracker" to="tld_fps"/>
    <remap from="image" to="$(arg image_topic)"/>
    <remap from="tracked_object" to="tld_tracked_object"/>
  </node>

  <!--include file="$(find tld_tracker)/launch/ros_tld_tracker.launch">
        <param name="image_topic" value="$(arg image_topic)" />
  </include-->

</launch>
